ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -80,6 +80,13 @@ def pytest_addoption(parser):
         default=False,
         help="Collect OCS logs when test case failed",
     )
+    parser.addoption(
```

Comment:
What's this code used for ?

Reply:
@ksandha This a new option for run-ci, which enables running IO in the background, while tests are running
See the PR description for this fixture purpose